### ISSUE TITLE
false-positive: expected-actual

### ISSUE THREAD

**Issue by ccoVeille** [Antonboom/testifylint#165]
[assert.Equal](https://pkg.go.dev/github.com/stretchr/testify/assert#Equal) 
> func Equal(t *testing.T, expected, actual)

Here is a piece of code

https://github.com/gofr-dev/gofr/blob/cd6a321af5bc90f465bd72edfe710e0d8268fff3/pkg/gofr/cmd_test.go#L63-L64

```go
assert.Equal(t, c.Request.Param("param"), "value")
assert.Equal(t, c.Request.Param("b"), "true")
```

Here is what testifylint suggests

```diff
-assert.Equal(t, "value", c.Request.Param("param"))
-assert.Equal(t, "true", c.Request.Param("b"))
+assert.Equal(t, c.Request.Param("param"), "value")
+assert.Equal(t, c.Request.Param("b"), "true")
```

**Comment by ccoVeille** [comment]
I'm tired. It works, the diff I got was when I reverted the code
